locksmith/migrations/20181212150352-create-lock.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
```

Comment:
This migration isn't particularly tightened for production, but should be adjustable within a few minutes when have a agreement that this general approach is a good path forward